lib-ee/emqx_license/src/emqx_license_http_api.erl

```diff
@@ -117,3 +142,40 @@ error_msg(Code, Msg) ->
     end;
 '/license'(post, _Params) ->
     {400, error_msg(?BAD_REQUEST, <<"Invalid request params">>)}.
+
+'/license/setting'(get, _Params) ->
+    {200, maps:remove(<<"key">>, emqx_config:get_raw([license]))};
+'/license/setting'(put, #{body := Setting}) ->
+    case emqx_license:update_setting(Setting) of
+        {error, Error} ->
+            ?SLOG(error, #{
+                msg => "bad_license_setting",
+                reason => Error
+            }),
+            {400, error_msg(?BAD_REQUEST, <<"Bad license setting">>)};
+        {ok, _} ->
+            ?SLOG(info, #{msg => "updated_license_setting"}),
+            '/license/setting'(get, undefined)
+    end.
+
+fields(key_license) ->
+    Key = lists:keyfind(key, 1, emqx_license_schema:fields(key_license)),
+    [
+        Key,
+        %% FIXME: remove when 5.2.0
+        {connection_low_watermark, #{
+            type => emqx_schema:percent(),
+            example => <<"75%">>,
+            deprecated => true,
+            desc => ?DESC(connection_low_watermark_field_deprecated)
+        }},
+        {connection_high_watermark, #{
+            type => emqx_schema:percent(),
+            example => <<"80%">>,
+            deprecated => true,
+            desc => ?DESC(connection_high_watermark_field_deprecated)
+        }}
```

Comment:
no need to keep them, there should be no compatibility issue.

Reply:
DONE